clients/pkg/promtail/targets/file/filetargetmanager.go

```diff
@@ -277,8 +324,23 @@ func (s *targetSyncer) sync(groups []*targetgroup.Group) {
 	s.droppedTargets = dropped
 }
 
-func (s *targetSyncer) newTarget(path string, labels model.LabelSet, discoveredLabels model.LabelSet) (*FileTarget, error) {
-	return NewFileTarget(s.metrics, s.log, s.entryHandler, s.positions, path, labels, discoveredLabels, s.targetConfig)
+func (s *targetSyncer) sendFileCreateEvent(event fsnotify.Event) {
```

Comment:
Nit: Could you add a doc string? If I understand correctly the event is sent to the matching file target, right?

Reply:
> If I understand correctly the event is sent to the matching file target, right?

Right.
OK, I'll add the description to this function.